src/zabapgit_git.prog.abap

```diff
@@ -151,6 +151,10 @@ CLASS lcl_git_pack DEFINITION FINAL FRIENDS ltcl_git_pack.
       EXPORTING ev_length TYPE i
       CHANGING  cv_data   TYPE xstring.
 
+    CLASS-METHODS zlib_compress_length
```

Comment:
can you rename this? eg. zlib_decompress? it does the actual decompression of the data